### PR TITLE
Create about section

### PR DESCRIPTION
**Pull request by vladanvi99**
In this pull request, I will add about section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <title>Portfolio</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" type="text/css" media="screen" href="./css/main.css">
+    <link rel="stylesheet" type="text/css" media="screen" href="./css/style.css">
```
Comment (jubaan) [inline index.html:9]: - [ ] [OPTIONAL] This line of code is calling a `style.css` file that doesn't exist. Let's try to maintain our code clean and remove this line.

File: css/main.css [hunk 2]
```diff
@@ -116,3 +117,272 @@ header nav .menu-btn:hover img {
...
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  gap: 12px;
+  padding-top: 36px;
+}
+
```
Comment (jubaan) [inline css/main.css:385]: - [ ]The `<main>` html tag is using a class `wrap` that is nowhere in the CSS file. Let's change this and create a new CSS selector and apply `display: grid` to it as is required by the project.

File: css/main.css [hunk 3]
```diff
@@ -116,3 +117,272 @@ header nav .menu-btn:hover img {
...
+#about .about-info .social-connect .social-links a:hover img {
+  -webkit-transform: scale(1.1) rotate(20deg);
+  transform: scale(1.1) rotate(20deg);
+  -webkit-transition: 0.3s;
+  transition: 0.3s;
+}
+
+#about .about-info .get-resume {
```
Comment (jubaan) [inline css/main.css:304]: - [ ] One brest practice as a developer is to create code that is reusable, CSS `id`s goes against this practice as it makes HTML and CSS harder to maintain over time and could cause unnecessary headaches. We should change all CSS `id`s to CSS `class` selectors, and update the elements in our HTML file with these changes.

This applies for all `id` in the CSS file.

File: index.html [hunk 4]
```diff
@@ -36,34 +37,210 @@ <h1>I’m Mariane Glad to see you!</h1>
...
+                        <a href="#">
                             <img src="images/redIcon.svg" alt="red-icon">
                         </a>
                     </li>
                 </ul>
             </div>
         </section>
         <!--BANNER END-->
+        <!-- WORKS START -->
+        <section id="works">
```
Comment (jubaan) [inline index.html:69]: - [ ] Let's edit the CSS rules that affect the projects section and apply a `display: grid` rule as it is one of the requirements of the project.

**Review by jubaan — CHANGES_REQUESTED** [review]
Hi @vladanvi99 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (vladanvi99) [reply index.html]: I fixed it, please check.

File: css/main.css (hunk 3 above)
Comment (vladanvi99) [reply css/main.css:304]: Also, I fixed and this, please check.

File: index.html (hunk 4 above)
Comment (vladanvi99) [reply index.html]: Parent of "my projects list" is a element with class="work-items", one line after class="works", I added display: grid; to that element.

File: index.html (hunk 1 above)
Comment (henatan99) [reply index.html]: Good to fix this! :clap:

File: css/main.css (hunk 3 above)
Comment (henatan99) [reply css/main.css]: - [ ] OPTIONAL: you have used CSS selectors like ul.work-items li.work-item. Please avoid this practice and just use the class selector itself like '.work-item' is enough.

Comment (vladanvi99) [reply css/main.css]: Okay, I will fix it.

File: index.html (hunk 4 above)
Comment (henatan99) [reply index.html]: This issue is addressed. The ul element which warps the projects with li tags has a display: grid CSS property. Nice work! :clap:

File: css/main.css (hunk 3 above)
Comment (vladanvi99) [reply css/main.css]: I fixed. Please check now.

File: index.html (hunk 4 above)
Comment (vladanvi99) [reply index.html]: Thank you :)

File: css/main.css [hunk 5]
```diff
@@ -97,22 +97,292 @@ header nav .menu-btn:hover img {
...
+  -webkit-transition: 0.3s;
+  transition: 0.3s;
+}
+
+.about {
+  margin-top: 114px;
+  background-color: #fff;
+  border-top-right-radius: 80px;
+  padding: 114px 24px;
+}
```
Comment (henatan99) [inline css/main.css:265]: - [ ] OPTIONAL: The about section should display child components with flex-box. This is not a mandatory requirement but this is important.

File: index.html (hunk 4 above)
Comment (vladanvi99) [reply index.html]: image
I can't submit the project yet.

**Review by henatan99 — APPROVED** [review]
Hi Vladan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Just consider the OPTIONAL comment I left in the inline comment when you get spare time. :smile:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**